components/dash-core-components/tests/integration/markdown/test_markdown.py

```diff
@@ -90,6 +91,278 @@ def test_mkdw002_dcclink(dash_dcc):
     )
 
     dash_dcc.start_server(app)
-    dash_dcc.percy_snapshot("mkdw002 - markdowns display")
+    assert dash_dcc.get_logs() == []
+
+
+@pytest.mark.parametrize("is_eager", [True, False])
+def test_mkdw003_without_mathjax(dash_dcc, is_eager):
+    app = Dash(__name__)
```

Comment:
```suggestion
    app = Dash(__name__, eager_loading=is_eager)
```
Which is presumably going to require changing the assertion below to:
```py
assert dash_dcc.driver.execute_script("return !!window.MathJax") == is_eager
```